tmt/options.py

```diff
@@ -124,7 +134,7 @@ def _check_method(self, context, args):
             if how is not None:
                 for method in methods:
                     if method.startswith(how):
-                        self._method = methods[method]
+                        self._method = methods[method]  # type: ignore
```

Comment:
Why this `ignore`?  `methods[method]` would be `click.core.Command`, IIUIC, `_method` has no annotation so far, could it be `Optional[click.core.Command]`?

Reply:
thanks, makes sense, fixed